### ISSUE TITLE
Discord crashes

### ISSUE THREAD

**Issue by PhilipWilson71** [RPM-Outpost/discord#8]
I have installed on centos 7 and see the following issue:
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: Feature redeclared; is this a duplicate flag?  voice_sound_stop_loop
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: Feature redeclared; is this a duplicate flag?  voice_relative_sounds
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: Feature redeclared; is this a duplicate flag?  voice_panning
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: Feature redeclared; is this a duplicate flag?  voice_multiple_connections
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: Feature redeclared; is this a duplicate flag?  media_devices
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: Feature redeclared; is this a duplicate flag?  media_video
Feb 28 14:40:16 693V2M2PW discord-stable.desktop: exception_ptr not yet implemented
Feb 28 14:40:16 693V2M2PW sedispatch: AVC Message for setroubleshoot, dropping message
Feb 28 14:40:16 693V2M2PW abrt-hook-ccpp: Process 28972 (Discord) of user 1005 killed by SIGABRT - ignoring (repeated crash)
Feb 28 14:40:16 693V2M2PW esets_daemon[5243]: warning[147b0000]: Cannot read from socket: Connection reset by peer
 Any ideas how to fix it?

**Comment by ObserverOfTime** [comment]
Is this issue still present? Can you paste the full error output (and please wrap it in ```)?

**Comment by TheElectronWill** [comment]
I've done some research on this.
It seems to be related to CentOS providing an old version of libcxx, which misses some features used by the recent Discord updates.

**Comment by likrtr** [comment]
Well, you can try installing libcxx-5.1 package from fedora, however it can be problematic.

**Comment by codermjb** [comment]
I have the same issue. I previously was using FlatPak, but sometimes there is no mic or audio. And the last update that came in broke the interface.

I have been trying to install natively and I also tried this repo and still no luck. I have the same issue with GLIBC. So I tried installing the Repo's below, but it's giving me an error saying it needs GLIBC 2.18 and when I try to do that it has another couple hundred dependencies.

sudo yum install http://dl.fedoraproject.org/pub/fedora/linux/updates/27/x86_64/Packages/l/libcxxabi-5.0.1-1.fc27.x86_64.rpm

sudo yum install http://dl.fedoraproject.org/pub/fedora/linux/updates/27/x86_64/Packages/l/libcxx-5.0.1-1.fc27.x86_64.rpm

**Comment by TheElectronWill** [comment]
@codermjb Have you tried [the package from rpmfind](https://www.rpmfind.net/linux/fedora/linux/updates/27/x86_64/Packages/l/libcxx-5.0.1-1.fc27.x86_64.rpm)?

**Comment by TheElectronWill** [comment]
I could try to download and install the libcxx rpm from rpmfind automatically, but I don't know what problems it might cause to other apps (or to the system itself)...

**Comment by codermjb** [comment]
@TheElectronWill I ended up reinstalling the Discord from Flatpak and it seems to be working in full now. Hopefully it continues to permanently work!

I wish Snap and Flatpak worked better though. They seem to be the future, but they have MASSIVE issues and nothing works as well, except Discord seems to be working decent.

**Comment by ObserverOfTime** [comment]
The rpmfind package should be the same as the fedoraproject one.

**Comment by TheElectronWill** [comment]
Nice! Flatpak isn't perfect (yet) but sometimes it's just so useful.

**Comment by codermjb** [comment]
@TheElectronWill I agree. I spent hours installing all my native apps and they worked perfectly. Tried out flatpak and reinstalled everything using FlatPak, I ran into so many issues with my apps, crashes when you click links, UI doesn't match theme, and opens multiple instances and doesn't display on Plank as 1 instance.

But, Flatpak at least makes unsupported native apps work, mainly Geary and Discord. I guess I can deal with that since it works and I need it for work. I have no other alternative. The only issue I think I have with Discord is links and sending files.
